alumni-stories/james-stewart-g3.md

```diff
@@ -0,0 +1,26 @@
+## Background
+
+Worked five years in politics - 3 for a Cabinet Minister in Westminster and 2.5 as a lobbyist for a charity. Only had experience in HTML/CSS coming into the course.
+
+## Since CodeClan
+Written: Sept 2018
+
+### First job
+
+My first position was for a very small start-up where I somehow landed the position as sole backend developer working with Node and Neo4j (wtf?). It was a really difficult post to hold for a first job, a baptism of fire. I had to really quickly learn how to do things without any support or input. There was no CTO. I relied on the CodeClan Alumni network and CodeYourFuture mentors to help me out (special mention to Scott Riley).
+
+I started looking for new jobs about a month into the role and left after 3 months because I got another job:
+
+### Current job
+
+I currently work for a start-up/scale-up as a Software Developer in a team of 5. The codebase is PHP and Vue.js - I had experience with Vue which was what attracted me to the post in the first place but no PHP experience.
+
+It's been night and day to my first job. I'm grateful that I had the experience of the first job as it helped me land this one, but I would really consider if taking a sole anything role is best for you when you are fresh out of CodeClan.
```

Comment:
`I would really consider if taking a sole anything role is best for you` 

Typo? 😄

Reply:
Nope! Sole front end, sole back end, sole DB person... all bad news!

Reply:
Oh I see - I'm sorry it wasn't clear but I understand now.